gpos/xml/scap/ssg/content/ssg-chainguard-gpos-ds.xml

```diff
@@ -222,11 +237,22 @@
                             For more information on inherited controls for containers see the DISA
                             Container Hardening Process Guide.</ns0:rationale>
                         <ns0:platform idref="cpe:/o:notapplicable" />
+        <ns0:ident system="http://cyber.mil/legacy">V-263653</ns0:ident>
+        <ns0:ident system="http://cyber.mil/legacy">SV-263653</ns0:ident>
+        <ns0:ident system="http://cyber.mil/cci">CCI-000366</ns0:ident>
+        <ns0:fixtext fixref="F-263653r1_fix">Configure the operating system to meet the requirement.</ns0:fixtext>
+        <ns0:fix id="F-263653r1_fix" />
```

Comment:
I'm not sure about the generic fix/remediation text, especially for the rules we've marked as not applicable, and thus have no check.